ruby/alias_manager.rb

```diff
@@ -0,0 +1,61 @@
+# 5.5 Solo Challenge: Manipulating Strings with Iteration
+
+#Release 0 _ pseudocode and write a method that takes a spy's real name 
+#(e.g., "Felicia Torres") and creates a fake name (alias):
+
+# 1) get spy's name
+# 2) Swapping the first and last name.
+# 3) change them all in lowercase
+# 4) split into letters
+# 5) Changing all of the vowels (a, e, i, o, or u) to the next vowel in 'aeiou', so "a" becomes "e"
+# 6) and all of the consonants to the next consonant in the alphabet. So 'd' would become 'f'.
+# 7) join them into string
+# 8) capitalize
+# 9) store and print
+
+Spy_Names = Hash.new
+
+def spy_name_generator
+	loop do
+		puts "Welcome to the Spy Name Generator.  Please type your full name. Type 'done' to exit."
+		real_name = gets.chomp.to_s
+		
```

Comment:
before any other logic check in input is done.  An outside method could be 

```
def check_done(input)
    input == "done"
end
```
run this for exit otherwise the rest of the code